lib/src/main/java/graphql/nadel/schema/NadelDirectives.kt

```diff
@@ -72,7 +75,7 @@ object NadelDirectives {
         )
         .inputValueDefinition(
             name = "value",
-            type = nonNull(GraphQLString),
+            type = nonNull(ExtendedScalars.Json),
```

Comment:
was this always wrong ???

Reply:
Nah, this used to only have "$source.blah" or "$args.blah" which is always a string.
We only need it now because the static types could be other types (mentioned in description of PR)